app/src/ui/diff/diff-mode.ts

```diff
@@ -13,6 +13,7 @@ const TokenNames: { [key: string]: string | null } = {
   '+': 'diff-add',
   '-': 'diff-delete',
   '@': 'diff-hunk',
+  ' ': 'diff-context',
```

Comment:
Should the token names be an `enum` with string values, so the type is `{ [key: string]: DiffMode | null }`?

Reply:
Yeah, most likely. Our DiffParser does something similar. I'm gonna say that it's out of scope for this PR though.